tests/test_ode.py

```diff
@@ -97,6 +97,40 @@ def test_polgar(self):
         # plt.legend()
         # plt.show()
 
+    def test_initial_parameters(self):
+        """
+        Identical to test_polgar, but with a0 as free Parameter.
+        """
+        a, b, c, d, t = variables('a, b, c, d, t')
+        k, p, l, m = parameters('k, p, l, m')
+
+        a0 = Parameter('a0', min=0, value=10)
+        b = a0 - d + a
+        model_dict = {
+            D(d, t): l * c * b - m * d,
+            D(c, t): k * a * b - p * c - l * c * b + m * d,
+            D(a, t): - k * a * b + p * c,
+        }
+
+        ode_model = ODEModel(model_dict, initial={t: 0.0, a: a0, c: 0.0, d: 0.0})
+
+        # Generate some data
+        tdata = np.linspace(0, 3, 1000)
+        # Eval
+        AA, AAB, BAAB = ode_model(t=tdata, k=0.1, l=0.2, m=.3, p=0.3, a0=10)
+        fit = Fit(ode_model, t=tdata, a=AA, c=AAB, d=BAAB)
+        results = fit.execute()
+        print(results)
+        self.assertAlmostEqual(a0.value, 10)
```

Comment:
I think the test should be a bit stronger. Perhaps give an initial guess of 11 instead of 10 to see if something really happened.

And add tests for `.params` and `.model_params` to see if they are what we expect them to be.